feature/home/presentation/home-ui/src/main/java/com/chocolatecake/ui/movie_details/MovieDetailsFragment.kt

```diff
@@ -124,14 +124,17 @@ class MovieDetailsFragment :
     private fun checkIsLoggedInOrNot() {
         val isLoggedIn = viewModel.state.value.isLogined
         if (isLoggedIn) {
-            showRatingBottomSheet()
+            showRatingBottomSheet(viewModel.state.value.movieUiState.voteAverage)
         } else {
             showSnackBar(getString(R.string.your_not_loged_in_to_rate))
         }
     }
 
-    private fun showRatingBottomSheet() {
+    private fun showRatingBottomSheet(voteAverage: Float) {
         val bottomSheet = RatingMovieBottomSheet()
+        bottomSheet.arguments = Bundle().apply {
+            putFloat("voteAverage", voteAverage)
+        }
```

Comment:
try to access the voteAverage from the bottom sheet using the viewModel instead of passing it throught bundle